tests/infrastructure/test_data_manager_stack.py

```diff
@@ -411,8 +424,16 @@ def test_indexer_lambda_iam_policy_resource_properties(template):
                             },
                         ],
                     },
+                    {"Action": "dynamodb:PutItem", "Effect": "Allow", "Resource": "*"},
                     {
```

Comment:
```suggestion
                    {
                        "Action": [
                            "secretsmanager:GetSecretValue",
                            "secretsmanager:DescribeSecret"
                        ],
                        "Effect": "Allow",
                        "Resource": {"Fn::Join": ["", ["arn:", {"Ref": "AWS::Partition"}, Match.string_like_regexp(":secretsmanager:.*:secret:.*")]]},
                    }
```

Reply:
I was able to get the test passing by replacing the secrets manager part with this. I got that format from writing the template json to a file and looking at it. I tried Matching the whole Resource entry, but that didn't work for some reason. I think it also didn't like that the GetSecretValue and DescribeSecret were out of order in that list. I'm not sure if I don't know what I'm doing or the cdk is just really annoying to test. But the code above should get the test passing and it seems to be what you were expecting. Let me know if you have any issues